todo_app/app.py

```diff
@@ -6,31 +6,34 @@
 from todo_app.data.trello import TrelloUtility
 from todo_app.data.model import ViewModel
 
-import os
-
+from dotenv import load_dotenv, find_dotenv
 
+import os
 
 def create_app():
     app = Flask(__name__)
 
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
```

Comment:
These lines currently break the selenium tests (they should really be in the `test_app()` method)

Reply:
Thanks for pointing that out Alex. Have removed those lines from `app.py`. These lines are already in `test_selenium.py` (lines 19 and 20)
 Made this change via https://github.com/kogoe1/DevOps-Course-Starter/pull/7 and have merged into `master`